index.js

```diff
@@ -23,10 +23,23 @@ function handleRelativePath(_relativePath) {
 }
 
 module.exports = walkSync;
-function walkSync (baseDir, _options, _relativePath) {
+function walkSync(baseDir, _options) {
+  var options = handleOptions(_options);
+
+  return _walkSync(baseDir, options).map(function(entry) {
+    return entry.relativePath;
+  });
```

Comment:
I think it's sensible to implement `walkSync` in terms of `entries`, but this particular implementation slows us down a bit: The [benchmark.js](https://github.com/joliss/node-walk-sync/blob/master/benchmark.js) timings for `walkSync` go from 280ms to 400ms on my system.

I seem to recall that `for` loops are faster than `.map` - maybe that contributes, or maybe there's some other way to optimize it. Do you want to take a look?

(I'm not opposed to losing a bit of performance, but since `walkSync` is so widely used, it might be worth trying to at least get close. Or if you prefer, we can merge now and optimize later.)